portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -54,13 +60,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     if (newComment.isEmpty()){
       response.sendError(HttpServletResponse.SC_BAD_REQUEST);
```

Comment:
error should be have descriptive message. 
`response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Comment must be a non-empty string.");`

Reply:
Lesson learned. Thanks!